test/jit/test_isinstance.py

```diff
@@ -51,6 +51,14 @@ def list_str_test(x: Any):
         x = ["1", "2", "3"]
         self.checkScript(list_str_test, (x,))
 
+    def test_list_tensor(self):
```

Comment:
Do we need a `test_list_tuple` that tests `Tensor` subtyping also?

Reply:
Yeah, added it.